cmd/lncli/commands.go

```diff
@@ -2070,6 +2071,23 @@ func parseChanPoint(s string) (*lnrpc.ChannelPoint, error) {
 	}, nil
 }
 
+func parseTimeLockDelta(timeLockDeltaStr string) (uint16, error) {
```

Comment:
Perhaps a short godoc here would be useful? In order to easily provide info on what the behavior is going to be e.g. with regards to `MaxTimeLockDelta`